spec/lib/govuk_healthcheck/checkup_spec.rb

```diff
@@ -63,6 +64,11 @@
     expect(response.dig(:checks, :critical_check, :status)).to eq(:ok)
   end
 
+  it "accepts Objects (can be initialized)" do
+    response = described_class.new([InitializedCheck.new(:param)]).run
```

Comment:
```suggestion
    response = described_class.new([TestHealthcheck.new]).run
```

You could use `TestHealthcheck.new` here instead, it'll test the same thing and we don't have to define the special `InitializedCheck` class.

Reply:
I have removed it, although I see value in setting a fixture that is close to what we are testing (or how it is used)